lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1829,6 +1837,28 @@ SPIRVValue *LLVMToSPIRV::transIndirectCallInst(CallInst *CI,
       BB);
 }
 
+SPIRVValue *LLVMToSPIRV::transAsmINTEL(InlineAsm *IA, SPIRVBasicBlock *BB) {
+  assert(IA);
+  StringRef TripleStr(M->getTargetTriple()); // TODO: Fix target
```

Comment:
Are you going to implement this todo? Could you please leave a comment what is wrong here?

Reply:
Problem is that if someone uses SPIRV translator, triple probably will be something like spirv64-unknown-unknown. What do we want here is real triple, i.e. one that this code actually came from. This is because inline assembler may vary from platform to platform and we want inform everyone about origin of inline assembly line.

Reply:
I meant that if you are not going to fix this in terms of this PR, then I suggest to leave some information within the code, so, someone else could pick up this TODO and fix it

Reply:
Done